### PR TITLE
Version cleanup

### PR DESCRIPTION
**Pull request by ps-jay**
Drop testing of Django 1.x - it's out of support
Drop Python 3.5 from `setup.py` - it's getting sec fixes only
Add Python 3.7 to `setup.py`
Add Python 3.8 to testing and `setup.py`

### CODE REVIEW

File: Dockerfile.testing [hunk 1]
```diff
@@ -42,113 +42,117 @@ RUN mkdir /src
 
 ##------------------------------------------------------------------------------
 FROM builder-base AS builder-py36
+ARG PYVER=36
+COPY --from=py36 /tmp/py${PYVER}-latest /tmp/
 
-RUN mkdir /opt/py36 && chown -R 3036 /opt/py36 /src
-USER 3036
+RUN mkdir /opt/py${PYVER} && chown -R 30${PYVER} /opt/py${PYVER} /src
+USER 30${PYVER}
```
Comment (alastairpat) [inline Dockerfile.testing:49]: This is… a thing 😆

File: README.rst [hunk 2]
```diff
@@ -307,6 +307,10 @@ Requirements: Docker 19.03.2 or newer and Docker Compose 1.24.1 or newer.
 Release History
 ---------------
 
+Version 1.9.1
+
+- State compatibility with 3.6, 3.7 and 3.8
```
Comment (alastairpat) [inline README.rst:312]: Maybe also state this version drops compatibility with Django 1?

File: Dockerfile.testing (hunk 1 above)
Comment (ps-jay) [reply Dockerfile.testing:49]: Yeah, seems extreme hey?
It's best not to compile things with the root user however.

File: README.rst (hunk 2 above)
Comment (ps-jay) [reply README.rst]: Done

**Pull request merged**